Makefile

```diff
@@ -109,7 +109,7 @@ wait:
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
 docker-build: generate ## Build docker image with the operator-controller.
-	docker build -t ${IMG} .
+	docker build -t ${IMG} -f Dockerfile ./bin/
 
 .PHONY: docker-push
 docker-push: ## Push docker image with the manager.
```

Comment:
WDYT about removing `docker-push`? We don't use it anywhere else, and it isn't part of the normal dev flow.